elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1220,6 +1221,7 @@ public Resource toResource(final Supplier<Map<String, Relationship>> relationshi
                 : dictionary.getId(obj));
         resource.setRelationships(relationshipSupplier.get());
         resource.setAttributes(attributeSupplier.get());
+        resource.setLinks(getResourceLevelLink());
```

Comment:
Hi @clayreimann @DeathByTape,
I'm back and want to finish this PR. 
I have a question about `PermitLinks` mode, think that I'm not quite sure how `url` should look like. I assume that the `url` might be something like `/people?links=true`. Do you have any thoughts on that?